### DISCUSSION TITLE
Move custom tags to separate module

### DISCUSSION THREAD

**Discussion by MekDrop** [ImpressCMS/impresscms#1050] [Ideas]
After we implement #207 I think we could move this functionality out of core to separate module.  I think I haven't used this functionality in many years. And this functionality needs improvements. I think it uses to much queries when it should be use some cached data.

**Comment by fiammybe** [comment]
Custom tags need improvement, but I wouldn't move it out of the core. The multilanguage functionality depends on it as far as I know.
There are many improvements possible, such as making the custom tags accept parameters, make them useful in all types of editors and also on site templates.

**Reply by MekDrop** [reply]
From what I can see in the code. Multilanguage has own text parser. Custom tags works as text sanitizer plugin. So, it's not very much related. So at least from this point it would be really possible to move into module.

And I agree that we could make custom tags more useful. If somebody would ask me to explain what that is, I probably will fail. If we decide to leave to functionality in the core I think we should do something about this problem.

**Comment by skenow** [comment]
They are a standard I use on almost every site - you can use them in templates and change them without having to edit files on the server.

https://www.impresscms.org/modules/simplywiki/index.php?page=Custom_Tags
